bots/pr/src/main/java/org/openjdk/skara/bots/pr/JEPCommand.java

```diff
@@ -0,0 +1,157 @@
+import org.openjdk.skara.issuetracker.Issue;
+import org.openjdk.skara.issuetracker.jira.JiraProject;
+
+import java.io.PrintWriter;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+public class JEPCommand implements CommandHandler {
+    static final String JEP_LABEL = "jep";
+    private static final String jepMarker = "<!-- jep: '%s' '%s' '%s' -->"; // <!-- jep: 'JEP-ID' 'ISSUE-ID' 'ISSUE-TITLE' -->
+    static final Pattern jepMarkerPattern = Pattern.compile("<!-- jep: '(.*?)' '(.*?)' '(.*?)' -->");
+    private static final String unneededMarker = "<!-- jep: 'unneeded' 'unneeded' 'unneeded' -->";
+
+    private void showHelp(PrintWriter reply) {
+        reply.println("""
+                Command syntax:
+                 * `/jep <issue-id>`
+                 * `/jep JEP-<jep-id>`
+                 * `/jep jep-<jep-id>`
+                 * `/jep unneeded`
+
+                Some examples:
+
+                 * `/jep JDK-1234567`
+                 * `/jep 1234567`
+                 * `/jep jep-123`
+                 * `/jep JEP-123`
+                 * `/jep unneeded`
+
+                Note:
+                The project prefix of the issue ID (`JDK-` in the above examples) is optional.
+                But the `JEP-` or `jep-` prefix is required when you provide a JEP ID.
+                And the issue type must be `JEP`.
+                """);
+    }
+
+    private Optional<Issue> getJepIssue(String args, PullRequestBot bot) {
+        Optional<Issue> jbsIssue;
+        if (args.startsWith("jep-") || args.startsWith("JEP-")) {
+            // Handle the JEP ID
+            jbsIssue = bot.issueProject().jepIssue(args.substring(4));
+        } else {
+            // Handle the issue ID
+            jbsIssue = bot.issueProject().issue(args);
+        }
+        return jbsIssue;
+    }
+
+    @Override
+    public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInstance, Path scratchPath, CommandInvocation command,
+                       List<Comment> allComments, PrintWriter reply, List<String> labelsToAdd, List<String> labelsToRemove) {
+        if (!pr.author().equals(command.user()) && !censusInstance.isReviewer(command.user())) {
+            reply.println("only the pull request author and [Reviewers](https://openjdk.java.net/bylaws#reviewer) are allowed to use the `jep` command.");
+            return;
+        }
+
+        var args = command.args().trim();
+        if (args.isEmpty() || args.isBlank()) {
+            showHelp(reply);
+            return;
+        }
+
+        var labelNames = pr.labelNames();
+        if ("unneeded".equals(args) || "uneeded".equals(args)) {
+            if (labelNames.contains(JEP_LABEL)) {
+                labelsToRemove.add(JEP_LABEL);
+            }
+            reply.println(unneededMarker);
+            reply.println("determined that the [JEP](https://openjdk.java.net/jeps) request " +
```

Comment:
```suggestion
            reply.println("determined that the JEP request " +
```

Reply:
Fixed and joined with next line and revised the corresponding tests.